config/locales/wishlist_items.en.yml

```diff
@@ -19,6 +19,7 @@ en:
   activerecord:
     attributes:
       wishlist_item:
+        domain_name: 'Domain name'
```

Comment:
I don't think we need to provide English values, since they are kinda translated by Rails itself:

irb(main):003:0> WishlistItem.human_attribute_name :domain_name
=> "Domeeninimi"
irb(main):004:0> I18n.locale
=> :et
irb(main):005:0> I18n.locale = :en
=> :en
irb(main):006:0> WishlistItem.human_attribute_name :domain_name
=> "Domain name"
irb(main):007:0>